assets/stylesheets/_components.scss

```diff
@@ -169,7 +169,6 @@
 @import 'components/progress-bar/style';
 @import 'components/pulsing-dot/style';
 @import 'components/purchase-detail/style';
-@import 'components/reader-main/style';
```

Comment:
What are the consequences of removing this?

Reply:
Yes! It's now loaded directly in the component:

https://github.com/Automattic/wp-calypso/pull/28272/files#diff-5e2375393afe7f203d69e6ca92bed475R17

This is part of the wider effort to migrate CSS build to Webpack - see https://github.com/Automattic/wp-calypso/pull/26820 and p4TIVU-98Q-p2.

Reply:
Ah, right! Totally missed that one.